rust/fastsim-core/src/vehicle.rs

```diff
@@ -37,6 +37,8 @@ lazy_static! {
 #[add_pyo3_api(
```

Comment:
@Romia87 , any arguments that get provided to `add_pyo3_api` _do not get compiled_ unless the [`fastsim-py`](https://github.com/NREL/fastsim/tree/fastsim-2/rust/fastsim-py) crate is being compiled because of [this feature](https://github.com/NREL/fastsim/blob/26edd86c100a03fc31586da2e93f5cfbd4fa4d63/rust/fastsim-core/proc-macros/src/add_pyo3_api.rs#L267).  As such, we want the pure rust version of the function to live elsewhere.  I'll drop a note on that next.